library/core/src/primitive_docs.rs

```diff
@@ -801,11 +801,57 @@ mod prim_array {}
 /// assert_eq!(2 * pointer_size, std::mem::size_of::<Box<[u8]>>());
 /// assert_eq!(2 * pointer_size, std::mem::size_of::<Rc<[u8]>>());
 /// ```
+///
+/// ## Trait Implementations
+///
+/// There are several implementations of common traits for slices. Some examples
+/// include:
+///
+/// * [`Clone`]
```

Comment:
Clone isn't implemented on `[T]` or `&[T]`  -- say more?

Reply:
Hmm good point. I just moved this content from `std::slice` docs. I'll remove this line.